pkg/sql/sqlbase/type_desc.go

```diff
@@ -253,26 +253,8 @@ func (desc *MutableTypeDescriptor) IsNew() bool {
 	return desc.ClusterVersion == nil
 }
 
-// AddEnumValue adds an enum member to the type.
+//AddEnumValue assumes that the type is an enum, and that the new value doesn't exist already in the enum
```

Comment:
make sure that your comment lines don't exceed 80 characters, and are full sentences (with a period at the end). You also must add a space after the `//`

Reply:
fixed